src/App.js

```diff
@@ -1,23 +1,87 @@
-import logo from './logo.svg';
+
 import './App.css';
+import React, {useState} from 'react';
+import contactData  from "./contacts.json"
 
 function App() {
+const [contacts, setContacts] = useState(contactData.slice(0,5));
+const randomContact = () =>{
```

Comment:
A good idea here is to apply Math.random only on the remaining contacts on the list.
The way you did you might get repeated contacts.